src/hooks/useLogDownloader/index.ts

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from "react";
+import { isProduction } from "utils/environmentVariables";
+import { leaveBreadcrumb } from "utils/errorReporting";
+/*
```

Comment:
nit. missing a star here, causing this not to be recognized as a JSDoc comment
```suggestion
/**
```